src/Containers/SettingsContainer.tsx

```diff
@@ -128,20 +161,38 @@ class SettingsContainer extends React.Component<Props, State> {
         );
     }
 
-    handleTestContact = async (contact: Contact) => {
-        try {
-            await this.props.moiraApi.testContact(contact.id);
-        } catch (error) {
-            this.setState({ error: error.message });
+    onRemoveContactBtnClick = async (contact: Contact) => {
+        const { settings } = this.state;
+        if (settings == null) {
+            throw new Error("InvalidProgramState");
         }
-    };
 
-    handleTestSubscription = async (subscription: Subscription) => {
-        const { moiraApi } = this.props;
+        const potentialyCrashedSubs =
```

Comment:
Исправил опечатку и нейминг:
```suggestion
        const potentiallyDisruptedSubscriptions =
```